test/resources.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+const blocked = require('..')
+const http = require('./cases/http')
+
+ blocked((time, stack, {type, resource}) => {
```

Comment:
Some assertions would be nice, to at least make it fail with a descriptive message when I remove the feature.